src/ensembl/src/header/launchbar/Launchbar.tsx

```diff
@@ -90,21 +92,33 @@ const Launchbar = () => {
               app="help"
               description="Help & documentation"
               icon={HelpIcon}
-              enabled={
-                isEnvironment([
-                  Environment.DEVELOPMENT,
-                  Environment.INTERNAL
-                ])
-              }
+              enabled={isEnvironment([
+                Environment.DEVELOPMENT,
+                Environment.INTERNAL
+              ])}
             />
           </div>
         </div>
       </div>
-      <div className={styles.about}>
-        <span className={styles.aboutText}>Genome data & annotation</span>
-      </div>
+      <AboutEnsembl />
     </div>
   );
 };
 
+// Temporarily disable the link to About Ensembl in production
+const AboutEnsembl = () =>
+  isEnvironment([Environment.DEVELOPMENT, Environment.INTERNAL]) ? (
+    <Link to="/about" className={styles.aboutEnsembl}>
+      About the
+      <Logotype className={styles.logotype} />
+      team & its work
```

Comment:
This bit is reused below too. Not necessary, but think it'll be nicer if we can carve this out into a separate node (like `AboutText`?) and use it in both the places.

Reply:
I think the "reuse" is only temporary since we dont have the help app/page on 2020. once this is available we will remove the code thats repeating completely. Does it then merit a separate node?

Reply:
I agree with Ridwan — when we start showing the link to the About page in production, we'll just remove `isEnvironment` with the component inside it, and the repetition will go away by itself.